sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/Snippets/Sample07_MockingEventProcessorHandlersTest.cs

```diff
@@ -0,0 +1,114 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Messaging.EventHubs.Processor;
+using Moq;
+using NUnit.Framework;
+using static System.Net.Mime.MediaTypeNames;
+
+namespace Azure.Messaging.EventHubs.Tests.Snippets
+{
+    [TestFixture]
+    public class Sample07_MockingEventProcessorHandlersTest
+    {
```

Comment:
Have separated the Event and Error Handler code

Reply:
This looks great!   It really cut down to just the focus area.